core/src/main/java/com/sequenceiq/cloudbreak/converter/JsonToOpenStackCredentialConverter.java

```diff
@@ -27,7 +27,13 @@ public OpenStackCredential convert(CredentialRequest source) {
         openStackCredential.setUserName(encryptor.encrypt(userName));
         String password = String.valueOf(source.getParameters().get(OpenStackCredentialParam.PASSWORD.getName()));
         openStackCredential.setPassword(encryptor.encrypt(password));
+        openStackCredential.setKeystoneVersion(String.valueOf(source.getParameters().get(OpenStackCredentialParam.KEYSTONE_VERSION.getName())));
```

Comment:
Validation improvement would be good:
* In case of v2 the tenantName is mandatory
* In case of v3 some of the new fields are mandatory